controllers/errors.controller.js

```diff
@@ -1,3 +1,18 @@
-exports.invalidPath = (req, res, next) => {
+exports.handleCustomErrors = (err, req, res, next) => {
+ if (err.status && err.msg) {
+  res.status(err.status).send({ msg: err.msg });
+ }
+ next(err);
+};
+
+exports.handlePSQLErrors = (err, req, res, next) => {
+ if (err.code === "22P02") {
+  res.status(400).send({ msg: "bad request" });
+ }
+ next(err);
+};
+
+exports.invalidPath = (err, req, res, next) => {
```

Comment:
an invalid path error handler only needs `req, res, next`, having `err` as well might mean express doesn't use it since it isn't actually detecting an error has occurred when the path is invalid